packages/framework/src/hooks/useRegisterBindings.ts

```diff
@@ -19,6 +19,9 @@ export const useRegisterBindings = <T extends { [key: string]: unknown }>(
   values: T,
   id?: string,
   methods?: InvokableMethods,
+  options?: {
+    forceRerender?: boolean;
+  },
```

Comment:
Let's rename this to `forceState` and make it check explicitly for `true` value. You're going to have to be careful with using this in your component.